code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -330,7 +330,7 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 /datum/game_mode/dynamic/proc/generate_advisory_level()
 	var/advisory_string = ""
 	if (prob(PULSAR_REPORT_CHANCE))
-		if (/datum/station_trait/bananium_shipment in SSstation.station_traits)
+		if(locate(/datum/station_trait/bananium_shipment) in SSstation.station_traits)
```

Comment:
this api is stupid

Reply:
i'm BYONDing out!

Reply:
might be nice to make `SSstation.has_station_trait` but oh well

Reply:
```suggestion
		if(HAS_TRAIT(SSstation, STATION_TRAIT_BANANIUM_SHIPMENTS))
```